docs/source/getting_started.rst

```diff
@@ -12,11 +12,11 @@ FESTIM requires FEniCS to run.
 
 FEniCS can be installed with Anaconda on MacOs and Linux. 
 In order to use the Anaconda distribution on Windows, use the `Windows Subsystem for Linux <https://learn.microsoft.com/en-us/windows/wsl/install>`_. 
-First `install Anaconda <https://docs.continuum.io/anaconda/install>`_ then run the following commands 
+First `install Anaconda <https://docs.continuum.io/anaconda/install>`_ then run the following commands::
 
-    conda create -n festim-env <br/>
-    source activate festim-env <br/>
-    conda install mamba <br/>
+    conda create -n festim-env
+    source activate festim-env
```

Comment:
Shouldn't it be conda activate instead?

Reply:
I mean thats what I put initially, can change it back